### DISCUSSION TITLE
icon for "blockquote" for a rich text editor

### DISCUSSION THREAD

**Discussion by charleneyjh1** [twilio-labs/paste#3313] [Request: Icon] [answered]
### Description

Hello, I need a "backquote" icon in a rich text editor I'm designing. Could we add this to Paste library? Thank you! 

Attaching a screenshot of the design that's at a very early stage. The "backquote" icon will show up with the list of icons in the rich text editor.

<img width="802" alt="Screenshot 2023-07-07 at 2 13 30 PM" src="https://github.com/twilio-labs/paste/assets/56697620/0a5523e4-f476-41ba-8724-eefcb5961942">

### Link to icon

https://www.streamlinehq.com/icons/core-line?search=backquote

### Icon type

- [X] UI icon
- [ ] Product icon

**Comment by SiTaggart** [comment]
Hi @charleneyjh1 

Just to double-check, do you mean "blockquote" or "backquote"? The link to streamline shows a "blockquote", which makes more sense to me as "backquote" or "backtick" is more common in code editors.

Once we are aligned on the right icon, you can either contribute yourself using the [contribution guidelines](https://paste.twilio.design/introduction/contributing/icons) or file a dependency

Thanks

**Reply by charleneyjh1** [reply]
oops sorry i made a typo. I meant blockquote. Thanks Simon!

**Reply by SiTaggart** [reply]
OK cool, great! That seems like a reasonable addition to the system. Would you be willing to follow the self-service guidelines and contribute?

**Reply by charleneyjh1** [reply]
yes let's do that! seems like the next step is a DSYS team member will be sending me a streamlined SVG

**Reply by loreina** [reply]
Hi @charleneyjh1, please reach back out to us when you're back in office and we can send over the svg file. Thanks!

**Reply by serifluous** [reply, marked as answer]
Marking as answered due to inactivity. Feel free to reply if you need more support!
